### ISSUE TITLE
FreeBSD Port

### ISSUE THREAD

**Issue by jgoffredo** [LuminanceHDR/LuminanceHDR#71]
FreeBSD port done.

Sent to FreeBSD.org and now is waiting for implementation in Ports Tree.

I suggest close this issue only after FreeBSD Ports Tree implementation.

**Comment by fcomida** [comment]
@jgoffredo good job. Did you make any change to the code? Open a pull request in that case, thank you.

**Comment by jgoffredo** [comment]
No changes. Compiled without problems.

Web Engine was a problem, but was ported to FreeBSD some days ago.

This is the second time I make a FreeBSD Port. In the first time I made some code changes. You can see my name "João Rocha Braga Filho" in FreeBSD patches and maintainers.

**Comment by fcomida** [comment]
Yes, thank you. Web Engine is still a problem for many Linux distros too, I made available a patch for reverting to WebKit, it seems you don't need it. Well done.

**Comment by jgoffredo** [comment]
Anti-ghosting is crashing always at the same point with the same images. It starts and crashes at 34% processing.

It stops processing at 34%, but memory usage continues to increase until crashes. It is, by now, the only problem I have found.

I don't know if happens in others systems.

I can send the files, if you want to test.

**Comment by fcomida** [comment]
I found another ag algorithm, need some spare time for implementing it, current on is ...crap anyway, I wouldn't bother too much. You can send me the files, let see what will happen here.
